specification/postgresql/resource-manager/readme.md

```diff
@@ -31,37 +31,13 @@ openapi-type: arm
 tag: package-flexibleserver-2025-08-01
 ```
 
-``` yaml $(package-singleservers)
-tag: package-2020-01-01
```

Comment:
Please put this back to resolve build errors. We have gotten approvals to move forward with Two Default Tag Error. Removing this introduces a new set of issues we are not prepared for until we deprecate older APIs.